packages/lib/modules/swap/SwapProvider.tsx

```diff
@@ -158,6 +158,9 @@ export function _useSwap({ poolActionableTokens, pool, pathParams }: SwapProvide
 
   if ((isTokenInSet && !tokenInInfo) || (isTokenOutSet && !tokenOutInfo)) {
     try {
+      if (isPoolSwap) {
```

Comment:
Maybe we need a special UI alert for this case but I will add it after deploying the live fix.